tests/functional/network_policy/conftest.py

```
@@ -0,0 +1,119 @@
+@pytest.fixture()
+def foreign_namespace(request):
+    """
+    Create a temporary namespace for negative (blocked traffic) testing.
+    Namespace is cleaned up after the test.
+
+    Returns:
+        OCP: Project object for the foreign namespace.
+    """
+    proj_obj = helpers.create_project(project_name="netpol-test")
+
+    def finalizer():
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `finalizer`

Add return type annotation: `None`

(ANN202)